apps/cow-tools/src/app/milkman/[network]/order/new/page.tsx

```diff
@@ -110,25 +119,26 @@ function TransactionCard({
   }, [formData]);
 
   async function handleOnSubmit(data: FieldValues) {
-    const decimals = 18; // TODO: BLEU-333
     const sellAmountBigInt = BigInt(
-      Number(data.tokenSellAmount) * 10 ** decimals,
+      Number(data.tokenSellAmount) * 10 ** data.tokenSell.decimals,
     );
     const priceCheckersArgs = priceCheckerInfoMapping[
       data.priceChecker as PRICE_CHECKERS
-    ].arguments.map((arg) => arg.convertInput(data[arg.name]));
+    ].arguments.map((arg) =>
+      arg.convertInput(data[arg.name], data.tokenBuy.decimals),
```

Comment:
Is this working for the UNI_V2 price checker? Since the `convertInput` of its `arguments` just receive one parameter

Reply:
https://www.loom.com/share/aacaf1489d9f41038e1ec80445dc4f19?sid=b48691ed-c4bf-4504-ab58-58f6ce0de285